configs/gotc/cromwell.conf.ctmpl

```diff
@@ -0,0 +1,410 @@
+      }
+    }
+  }
+ }
+
+backend {
+  default = "PAPIv2"
+  providers {
+    PAPIv2 {
+      actor-factory = "cromwell.backend.google.pipelines.v2alpha1.PipelinesApiLifecycleActorFactory"
+      config {
+        slow-job-warning-time: 24 hours
+
+        // Google project
+        project = {{ if eq $env "prod" }}"broad-gotc-{{$env}}"{{else}}"broad-exomes-dev1"{{end}}
+
+        // Base bucket for workflow executions
+        root = "gs://broad-gotc-{{$env}}-cromwell-execution"
+
+        # Set this to the lower of the two values "Queries per 100 seconds" and "Queries per 100 seconds per user" for
+        # your project.
+        #
+        # Used to help determine maximum throughput to the Google Genomics API. Setting this value too low will
+        # cause a drop in performance. Setting this value too high will cause QPS based locks from Google.
+        # 1000 is the default "Queries per 100 seconds per user", 50000 is the default "Queries per 100 seconds"
+        # See https://cloud.google.com/genomics/quotas for more information
+        genomics-api-queries-per-100-seconds = 50000
+
+        # Polling for completion backs-off gradually for slower-running jobs.
+        # This is the maximum polling interval (in seconds):
+        maximum-polling-interval = 600
+
+        # This limit is based on empirical measures of Cromwell's throughput. Identical to FireCloud as of 2019-09-04
+        concurrent-job-limit = 14400
+
+        # In FireCloud private Docker Hub configuration values will be provided in workflow options only.
+        dockerhub { }
+
+        genomics {
+          # A reference to an auth defined in the `google` stanza at the top.  This auth is used to create
+          # Pipelines and manipulate auth JSONs.
+          auth = "service-account"
+
+          // alternative service account to use on the launched compute instance
+          // NOTE: If combined with service account authorization, both that serivce account and this service account
+          // must be able to read and write to the 'root' GCS path
+          compute-service-account = "default"
+
+          # Endpoint for APIs, no reason to change this unless directed by Google.
+          endpoint-url = "https://genomics.googleapis.com/"
+
+          # Restrict access to VM metadata. Useful in cases when untrusted containers are running under a service
+          # account not owned by the submitting user
+          restrict-metadata-access = false
+        }
+        batch-requests {
+          timeouts {
+            read = 120 seconds
+            connect = 120 seconds
+          }
+        }
+        filesystems {
+          gcs {
+            # A reference to a potentially different auth for manipulating files via engine functions.
+            auth = "service-account"
+          }
+          drs {
+            auth = "service-account"
+          }
+          http {}
+        }
+      }
+    }
+  }
+}
+
+services {
```

Comment:
Don't we need `KeyValue` and `LoadController` services in GOTC?

Reply:
They're both already included, by default, in the reference.conf.

https://github.com/broadinstitute/cromwell/blob/45/core/src/main/resources/reference.conf#L483-L524

Reply:
This won't erase anything in the `reference.conf` so the defaults there should still apply